qa/automation/CareerPortal.Tests.Scenarios/3. Send_CV/Send_CV.feature

```diff
@@ -0,0 +1,148 @@
+    And writed a message 500 characters long
+    When user clicks on a button "Send"
+    Then CV should be successfully submitted
+
+    Examples:
+        | name         | email                    |
+        | Egor  Berdin | egor.berdin@quantori.com |
+
+
+
+Scenario Outline: CV submission without checked the checkbox
+    Given user entered a <name>, <email>, <message>, attached CV (PDF less than 10 MB)
+    When user clicks on a button "Send"
+    Then the message next to the checkbox will turn red
+
+    Examples:
+        | name         | email                    | message     |
+        | Egor  Berdin | egor.berdin@quantori.com | Hello_world |
+
+
+
+Scenario Outline: CV submission without CV
+    Given user entered a <name>, <email>, <message>, checked the checkbox
+    When user clicks on a button "Send"
+    Then the button message "Attach CV" will turn red
+
+    Examples:
+        | name         | email                    | message     |
+        | Egor  Berdin | egor.berdin@quantori.com | Hello_world |
+
+
+
+Scenario Outline: CV submission with CV in different formats
+    Given user clicked on the button "Attach CV"
+    When user selects a <format> file
+    Then <result> 
+
+    Examples:
+        | format | result                                          |
+        | .txt   | there should be a warning "Invalid file format" |
+        | .doc   | the file will be added successfully             |
+        | .docx  | the file will be added successfully             |
+        | .rtf   | the file will be added successfully             |
+        | .pdf   | the file will be added successfully             |
+        | .zip   | there should be a warning "Invalid file format" |
+        | .rar   | there should be a warning "Invalid file format" |
+        | .avi   | there should be a warning "Invalid file format" |
+        | .jpg   | there should be a warning "Invalid file format" |
+        | .png   | there should be a warning "Invalid file format" |
+
+
+
+Scenario Outline: CV submission with CVs of different sizes
+    Given user entered a valid name, email, message, checked the checkbox
+    When user selects a .pdf <size> Mb file
+    Then <result>
+
+    Examples:
+        | size | result                                          |
+        | 5    | the file will be added successfully             |
+        | 9,9  | the file will be added successfully             |
+        | 10   | the file will be added successfully             |
+        | 10,1 | there should be a warning "Max file size 10 Mb" |
+
+
+
+Scenario Outline: CV submission with invalid Name format
+    Given user entered a <name>, <email>, <message>, attached CV (PDF less than 10 MB), checked the checkbox
+    When user clicks on a button "Send"
+    Then under the field "Name" a warning appears "Invalid name format"
+
+    Examples:
+        | name | email                    | message     |
+        | Egor | egor.berdin@quantori.com | Hello_world |
+
+
```

Comment:
Additional cases: 

1. can we attach several files ? Or what happened if you try attach the second file ? 
2. what about limit characters of Name/Email fields?